tests/xtro-sharpie/macOS-WebKit.ignore

```diff
@@ -2,3 +2,46 @@
 
 # TODO WKWebView should conform to NSTextFinderClient is marked as macos(WK_MAC_TBA) as of 11.4 beta3
 !unknown-native-enum! WKSelectionGranularity bound
+
+# xcode 15
+# API has been deprecated in macOS..
+!incorrect-protocol-member! WebOpenPanelResultListener::cancel is REQUIRED and should be abstract
+!incorrect-protocol-member! WebOpenPanelResultListener::chooseFilename: is REQUIRED and should be abstract
+!incorrect-protocol-member! WebOpenPanelResultListener::chooseFilenames: is REQUIRED and should be abstract
+!incorrect-protocol-member! WebPolicyDecisionListener::download is REQUIRED and should be abstract
+!incorrect-protocol-member! WebPolicyDecisionListener::ignore is REQUIRED and should be abstract
+!incorrect-protocol-member! WebPolicyDecisionListener::use is REQUIRED and should be abstract
+
+# older/legacy API
+!missing-selector! +NSObject::isKeyExcludedFromWebScript: not bound
```

Comment:
Support for these API were added as part of macOS 10+ and is noted as legacy on Apple Docs..do we want to add support for these right now?
https://developer.apple.com/documentation/objectivec/nsobject/webplugincontainer?language=objc

Reply:
Sounds good to not do anything for now.